Core/WKWebViewExtension.swift

```diff
@@ -67,6 +67,7 @@ extension WKWebView {
         dataStore.fetchDataRecords(ofTypes: allData) { records in
             let externalRecords = records.filter { $0.displayName != Constants.cacheKey }
```

Comment:
Definitely. We'd pass in a array of items to blow away based on top level domains we hit during a page load. All webkit exposes for matching is this `displayName` so it probably won't be a perfect match, but we'll hopefully catch most items added to the cache.